.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright The Ratify Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# template created based on verbiage from https://github.com/oras-project/oras/blob/cc40e430ee0cc124250f9855a2e99f535f759221/.github/ISSUE_TEMPLATE/bug-report.yaml
+name: Bug Report
+description: File a bug report
+labels: [bug, triage]
+body:
+  - type: markdown
+    id: preface
+    attributes:
+      value: |
+        Thank you for reporting bugs to ratify-go!
+  - type: textarea
+    id: environment
+    validations:
+      required: true
+    attributes:
+      label: What happened in your environment? 
```

Comment:
If you take a look at https://github.com/notaryproject/notation-go/blob/main/.github/ISSUE_TEMPLATE/bug-or-issue.yaml, you will find that it misses "How can we reproduce it".

Reply:
Added.